legacy/README.md

```diff
@@ -43,10 +43,6 @@ This will require you to use:
 - the [`portify`](https://github.com/guardian/commercial-templates/blob/master/src/_shared/js/dev.js) method in `src/_shared/js/dev.js` to programmatically switch between 7000 and 9000 when making outbound requests.
 - the [`JsonComponent`](https://github.com/guardian/frontend/blob/master/common/app/common/JsonComponent.scala) method to serve the JSON response from the frontend controller. This method wraps the JSON response in a [CORS](https://github.com/guardian/frontend/blob/master/common/app/model/Cors.scala) header which allows cross-origin requests.
 
-### Automated ads
-
-A number of ads (`events-multiple` and `masterclass-multiple`) use a reader revenue endpoint. In their associated `test.json` files you will see that the URL's refer to `mem.thegulocal.com` as the domain. The `_src/js/events-multiple.js` needs to be updated to reflect this domain also. This requires the [`membership-frontend`](https://github.com/guardian/membership-frontend) to be running and using the `nginx` server. Make sure to set this up before testing these ads.
-
```

Comment:
I spoke to Emma regarding this and she said we can remove this section as `events-multiple` is migrated to Svelte and `masterclass-multiple` have been shut down.